### ISSUE TITLE
Update/fix simple-matrix-bot-lib

### ISSUE THREAD

**Issue by HarHarLinks** [matrix-org/matrix.org#1256]
@KrazyKirby99999 this page is not up to date:

https://github.com/matrix-org/matrix.org/blob/49db2160c3742a4656282afaa1c6d5371b581799/gatsby/content/projects/sdks/simple-matrix-bot-lib.mdx#L20

I suggest simply removing the version here, what purpose does it serve?

https://github.com/matrix-org/matrix.org/blob/49db2160c3742a4656282afaa1c6d5371b581799/gatsby/content/projects/sdks/simple-matrix-bot-lib.mdx#L27

This relative link leads to nowhere.

**Comment by KrazyKirby99999** [comment]
You are correct, I think that it should be replaced by a shorter example along with generic links to documentation, examples and this repo.
